src/glue/jobs/compare_parquet_datasets.py

```diff
@@ -0,0 +1,422 @@
+import os
+import json
+import logging
+import argparse
+
+import boto3
+import pandas as pd
+import synapseclient
+from pyarrow import fs
+import pyarrow.parquet as pq
+
+# from json_to_parquet import INDEX_FIELD_MAP
+
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+
+INDEX_FIELD_MAP = {
```

Comment:
Note that this will get removed if we decide to move this `INDEX_FIELD_MAP` to a common script or just use the one in `json_to_parquet` but for ease of testing (because would have to do it in a docker image due to json_to_parquet requiring glue), it seemed easier to leave this in.

Reply:
This redundancy is fine. I don't think it's as simple to remove as including a config.py with this constant in the same directory, since these are glue job scripts that are stored in S3.